env0/resource_drift_detection.go

```diff
@@ -2,6 +2,7 @@ package env0
 
 import (
 	"context"
+
```

Comment:
Why empty line

Reply:
It come up when I am save the file, tried to fix it but then failed in fmt test job

Reply:
@samuel-br 👍